hx_lti_initializer/views.py

```diff
@@ -369,7 +371,8 @@ def course_admin_hub(request):
             'debug': debug,
             'starter_object': starter_object,
             'starter_object_id': object_id,
-            'starter_collection_id': collection_id
+            'starter_collection_id': collection_id,
+            'utm_source': request.session.session_key if not request.session['is_staff'] else '',
```

Comment:
That's a good point. This issue won't actually impact our canvas instance since we're running on the same domain (*.harvrad.edu), so we don't really have a need for it on the admin hub.

I guess the concern that I have in mind is partially implementing cookieless-sessions and then having to know where it will work and won't work, instead of just being able to rely on a complete solution (i.e. parity with cookie-based sessions middleware). I think we can mitigate this concern with a little documentation, which I'll volunteer to do (working on a revised README as we speak). 

I was thinking about this earlier and a more complete solution would probably involve monkey-patching the `reverse()` function and `url()` templatetag so they automatically add the sessionid to the URL for cookieless sessions, and then of course add support in the JS. But that's a bigger task than what's already implemented, so I think just documenting the limitations of the current solution would be enough.